.github/workflows/release-launch-agent.yml

```diff
@@ -0,0 +1,67 @@
+name: Release Launch Agent
+
+defaults:
+  run:
+    working-directory: ./tools/launch_release/release
+
+on:
+  workflow_dispatch:
+    inputs:
+      preReleaseString:
```

Comment:
nit: but i think they use snake_case for the inputs

Reply:
This is contained within the revert. I'm going to leave as is, since it conforms with internal documentation that I'd prefer not to have to update for a sunsetting feature